deployer/deployer-services/src/main/java/io/hyscale/deployer/services/broker/K8sResourceBroker.java

```diff
@@ -61,29 +61,29 @@ public void withNamespace(String namespace) {
         }
         V1ObjectMeta objectMeta = kubernetesResource.getV1ObjectMeta();
         T obj = (T) kubernetesResource.getResource();
-        String namespace = kubernetesResource.getV1ObjectMeta().getNamespace();
+        String resourceNamespace = kubernetesResource.getV1ObjectMeta().getNamespace();
```

Comment:
Why this change ?

Reply:
In class we already have a namespace field, which as per sonar analysis is a major code smell.